src/Controllers/DepartmentController.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: elena
+ * Date: 06.11.16
+ * Time: 0:49
+ */
+
+namespace Controllers;
+
+use Repositories\UniversityRepository;
+use Repositories\DepartmentRepository;
+use Repositories\StudentsRepository;
+use Repositories\DisciplinesRepository;
+use Repositories\TeacherRepository;
+use Repositories\HomeworkRepository;
+
+class DepartmentController
+{
+    private $resultsDepartment;
+
+    private $resultsUniversity;
+
+    private $resultsStudents;
+
+    private $resultsDisciplines;
+
+    private $resultsTeacher;
+
+    private $resultsHomework;
+
+    private $loader;
+
+    private $twig;
+
+    /**
+     * DepartmentController constructor.
+     * @param $connector
+     */
+    public function __construct($connector)
+    {
+        $this->resultsDepartment = new DepartmentRepository($connector);
+        $this->resultsUniversity = new UniversityRepository($connector);
+        $this->resultsStudents = new StudentsRepository($connector);
+        $this->resultsDisciplines = new DisciplinesRepository($connector);
+        $this->resultsTeacher = new TeacherRepository($connector);
+        $this->resultsHomework = new HomeworkRepository($connector);
+        $this->loader = new \Twig_Loader_Filesystem('src/Views/templates/');
+        $this->twig = new \Twig_Environment($this->loader, array(
+            'cache' => false,
+        ));
+    }
+
+    /**
+     * @return string
+     */
+    public function indexAction()
+    {
+        $resultsDataUniversity = $this->resultsUniversity->findAll(1000, 0);
```

Comment:
I don't understand, if it is the DepartmentController::indexAction - this action needs only departments data. Why do you query all your tables? Do you output all this data in your template? Remove not necessary queries please